local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
This file is probably not supposed to be uploaded to git (and github). Do check with your TA or prof about this.